yt/frontends/boxlib/data_structures.py

```diff
@@ -1405,6 +1406,14 @@ def _read_header(raw_file, field):
                 lo_corner, hi_corner, node_type = _line_to_numpy_arrays(clean_line)
                 boxes.append((lo_corner, hi_corner, node_type))
 
+            if nghost is None:
+                try:
+                    nghost = np.array(len(lo_corner) * [ng])
+                except NameError:
+                    # If lo_corner is not defined.
+                    # Will this ever happen?
```

Comment:
I don't think it's worth catching an exception if we don't even know why it would happen.